backend/core/quivr_core/models.py

```diff
@@ -8,6 +7,7 @@
 from langchain_core.pydantic_v1 import BaseModel as BaseModelV1
 from langchain_core.pydantic_v1 import Field as FieldV1
 from pydantic import BaseModel
+from quivr_api.modules.knowledge.dto.inputs import KnowledgeStatus
```

Comment:
quivr_api depends on quivr_core we can't have imports from quivr_api in quivr_core